src/PrestaShopBundle/Resources/config/services/bundle/mail.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.service.mail.mail_template_folder_catalog:
+    class: 'PrestaShopBundle\Service\Mail\MailTemplateFolderCatalog'
+    arguments:
+      - '%mail_themes_dir%'
+
+  prestashop.service.mail.mail_template_parameters_builder:
+    class: 'PrestaShopBundle\Service\Mail\MailTemplateParametersBuilder'
+    arguments:
+      -
+        emailPublicWebRoot: "@=service('prestashop.adapter.legacy.context').getRootUrl()"
```

Comment:
if you use this syntax, do it everywhere or nowhere :+1: 

( as most of the files doesn't use it, I'm in favor of "nowhere" but because it's a new file, it's up to you)

Reply:
this is not the new service syntax where you specify the arguments names, the first parameter is an associative array
this is equivalent to

```
   arguments:
      - [emailPublicWebRoot: "@=service('prestashop.adapter.legacy.context').getRootUrl()"]
```